### PR TITLE
Copy update on /telco/osm

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Copy update on `/telco/osm`

## QA

- go to `/telco/osm` or https://ubuntu-com-11611.demos.haus/telco/osm
- Compare page against [copy doc](https://docs.google.com/document/d/1LQim0WRVdDZE9-ToFKSD52O7kgurMoytrgDDAPYnug8/edit#)

## Issue / Card

Fixes [#5323](https://github.com/canonical-web-and-design/web-squad/issues/5323)

### CODE REVIEW

File: templates/telco/osm/index.html [hunk 1]
```diff
@@ -237,17 +238,25 @@ <h2>OSM Community</h2>
...
-        <a href="https://charmed-osm.com/osm-hackfests">Discover our contributions</a>
+        <a href="https://charmed-osm.com/osm-hackfests">Discover our contributions&nbsp;&rsaquo;</a>
       </p>
     </div>
   </div>
 </section>
+<section class="p-strip is-bordered is-deep">
+  <div class="u-fixed-width">
+    <h2>Try OSM today</h2>
+    <p>Follow the <a href="https://www.youtube.com/watch?v=DJhtjnWdfi8">video</a> of installation or <a href="https://juju.is/tutorials/charmed-osm-get-started#1-introduction">tutorial</a>. If you wish to understand the whole architecture before hands on please watch this <a href="https://www.youtube.com/watch?v=3EWjL6pIXe0">video</a>.</p>
```
Comment (mtruj013) [inline templates/telco/osm/index.html:255]: The wording here is a little weird. "Follow the video of installation..." isn't technically grammatically incorrect as far as I know, but "Follow the installation video..." sounds a lot better I think. Also, "hands on" is an adjective so the sentence `If you wish to understand the whole architecture before hands on please watch this video` should be something like `If you wish to understand the whole architecture before getting hands on experience please watch this video.` I'll leave comments in the doc

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Looks good @minkyngkm! Just a few comments in the code and this logo section needs a spacing fix for that last logo
Screenshot 2022-05-16 at 14 06 19

**Comment by minkyngkm** [comment]
Thanks @mtruj013 for reviewing! It's been all updated! 👍

File: templates/telco/osm/index.html [hunk 2]
```diff
@@ -197,18 +195,21 @@ <h2>Enterprise support</h2>
...
         <li class="p-list__item is-ticked">Provides MANO stack enterprise support.</li>
         <li class="p-list__item is-ticked">Offers fully-managed OSM.</li>
-        <li class="p-list__item is-ticked">Includes one year of bug fixes and two years of security updates.</li>
+        <li class="p-list__item is-ticked">Includes two years of bug fixes and security updates.</li>
       </ul>
       <p>
         <a href="/legal/ubuntu-advantage-service-description">Ubuntu Support&nbsp;&rsaquo;</a>
       </p>
+      <p>
+        <a href="/legal/ubuntu-advantage-service-description">Community Support&nbsp;&rsaquo;</a>
```
Comment (mtruj013) [inline templates/telco/osm/index.html:204]:
```suggestion
        <a href="https://osm.etsi.org/news-events/blog/72-etsi-osm-ten-declared-as-the-first-lts-version-of-open-source-mano">Community Support&nbsp;&rsaquo;</a>
```

File: templates/telco/osm/index.html [hunk 3]
```diff
@@ -237,17 +238,25 @@ <h2>OSM Community</h2>
...
+        <a href="https://charmed-osm.com/osm-hackfests">Discover our contributions&nbsp;&rsaquo;</a>
       </p>
     </div>
   </div>
 </section>
+<section class="p-strip is-bordered is-deep">
+  <div class="u-fixed-width">
+    <h2>Try OSM today</h2>
+    <p>Follow the installation <a href="https://www.youtube.com/watch?v=DJhtjnWdfi8">video</a> or <a href="https://juju.is/tutorials/charmed-osm-get-started#1-introduction">tutorial</a>. If you wish to understand the whole architecture before getting hands on experience please watch this <a href="https://www.youtube.com/watch?v=3EWjL6pIXe0">video</a>.</p>
+    <p>We also provide installation of <a href="https://www.youtube.com/watch?v=5iURQB8cLJ0">OSM in a box</a> with the minimum possible resources needed to be perfectly fucntional.</p>
```
Comment (mtruj013) [inline templates/telco/osm/index.html:256]:
```suggestion
    <p>We also provide installation of <a href="https://www.youtube.com/watch?v=5iURQB8cLJ0">OSM in a box</a> with the minimum possible resources needed to be perfectly functional.</p>
```

**Review by mtruj013 — APPROVED** [review]
LGTM! Just a couple of minor things I missed:

**Pull request merged**